user_templates/mcu_defaults/main/overrides/BTT_SKR_Pico_v1.0_overrides.cfg

```diff
@@ -0,0 +1,27 @@
+
+#-------------------------#
+#   MCU board overrides   #
+#-------------------------#
+
+# Those overrides are automatically added when you select a mcu board with specific configuration options like shared pins.
+
+[tmc2209 stepper_x]
+uart_pin: XYZE_TMCUART
+tx_pin: XYZE_TMCTX
+uart_address: 0
+
+[tmc2209 stepper_y]
+uart_pin: XYZE_TMCUART
+tx_pin: XYZE_TMCTX
+uart_address: 2
+
+[tmc2209 stepper_z]
+uart_pin: XYZE_TMCUART
+tx_pin: XYZE_TMCTX
+uart_address: 1
+
+# comment this section if you are using a toolhead with own mcu
+[tmc2209 extruder]
+uart_pin: XYZE_TMCUART
+tx_pin: XYZE_TMCTX
+uart_address: 3
```

Comment:
And this is only pins definition finally. So this is why I think it can be moved to mcu.cfg